pandas/core/frame.py

```diff
@@ -1143,33 +1142,57 @@ def to_gbq(self, destination_table, project_id, chunksize=10000,
 
         Parameters
         ----------
-        dataframe : DataFrame
-            DataFrame to be written
```

Comment:
this needs to stay

Reply:
The validation script was complaining about this one. I think because this is a method of `DataFrame`, so there is no `dataframe` argument. (I also tried `self` and it complained about that, too.)

```
################################################################################
################################## Validation ##################################
################################################################################

Errors found:
	Errors in parameters section
		Unknown parameters {'dataframe'}
		Parameter "kwargs" description should finish with "."
	No returns section found
	Missing description for See Also "pandas_gbq.to_gbq" reference
	No examples section found
```

Reply:
oh don’t worry about that too much

Reply:
I think @tswast is correct that this should be removed. The method does not take a dataframe as input (it is writing `self` to gbq)